src/Components/test/testassets/BasicTestApp/DuplicateAttributesComponent.razor

```diff
@@ -0,0 +1,5 @@
+<div id="duplicate-on-component">
+    <DuplicateAttributesChildComponent Message="test1" Message="test2" />
```

Comment:
I'm hoping we don't need this feature. Duplicate attributes are only useful in conjunction with splatting - I think we can consider it either erroneous or undefined-behavior to have duplicate explicit (non-splatted) attributes. Then we can avoid most of the extra runtime costs.

Do you think that would be reasonable?

Reply:
Discussed offline and fixed in the tooling repo.